Documentation changes for the 2FA additions (#40411)
keycloak/keycloak: docs/documentation/server_admin/topics/authentication/recovery-codes.adoc

. Click *Authentication* in the realm menu.
. Click the *Browser* flow.
. Locate the execution *Recovery Authentication Code Form* inside the *Browser - Conditional 2FA* sub-flow.
. Change the _requirement_ from _Disabled_ to _Alternative_ for that execution.
+
.Recovery Codes Browser flow
image:images/recovery-codes-browser-flow.png[Recovery Codes Browser flow]
+

> With this configuration, both 2FA authenticators (`OTP Form` and `Recovery Authentication Code Form`) are alternate ways to log into {project_name}. If the user has configured both credential types, the credential with the highest priority will be displayed by default, but another option *Try Another Way* will be available that allows to select the other 2FA method.

Reviewer: ```suggestion
With this configuration, both 2FA authenticators (`OTP Form` and `Recovery Authentication Code Form`) are alternate ways to log into {project_name}. If the user has configured both credential types, the credential with the highest priority will be displayed by default, but the *Try Another Way* option will appear so you can select another 2FA method.
```
